docs/using/recipes.md

```diff
@@ -0,0 +1,256 @@
+# Contributed Recipes
+
+Users sometimes share interesting ways of using the Jupyter Docker Stacks. We encourage users to [contribute these recipes](../contributing/recipes.html) to the documentation in case they prove useful to other members of the community. The sections below capture this knowledge.
+
+## Add RISE
+
+@pdonorio said:
```

Comment:
Is @ messaging going to properly cite when the docs are built in RST?

Reply:
Probably not. It didn't on the wiki either, surprisingly: https://github.com/jupyter/docker-stacks/wiki/Docker-recipes

Reply:
Do we want to make this a link instead then?  Are you planing on keeping this in MD or converting to RST for the website?